platform/mellanox/iproute2/patch/0001-Remove-arpd-from-iproute2-package.patch

```diff
@@ -0,0 +1,56 @@
+From 463a11c59ac668e038d22c7623f1bdbc3dc5012b Mon Sep 17 00:00:00 2001
+From: liora <liora@contoso.com>
```

Comment:
Can you please make sure patch appears with correct email? This may be because you created from inside slave docker.

Reply:
Done.